### PR TITLE
[s] Shit lets fix conflicts on a year old pull

### PR DESCRIPTION
**Pull request by Incoming5643**
:dancer: #8679 :dancer:

### CODE REVIEW

**Comment by TheRealGLH** [comment]
Needs lizards too, the last couple of times I had the """"""""privilege"""""""" of watching ERP on /tg/ was with lizard players.

**Comment by Incoming5643** [comment]
You don't need Tactical ERPers for a race that just does that shit without provocation anyways

**Comment by Zilenan91** [comment]
The difference is that regular lizards aren't as F A B U L O U S as these would be

**Comment by Incoming5643** [comment]
You will have to live with the ramifications of what you've requested

d10_icon

---

M E R G E  R E A D Y

**Comment by Zilenan91** [comment]
Oh wow I thought it was something like Jojo not this but this is also fine

**Comment by PKPenguin321** [comment]
>You will have to live with the ramifications of what you've requested

jesus christ how terrifying

File: code/modules/admin/verbs/one_click_antag.dm [hunk 1]
```diff
@@ -1,590 +1,662 @@
...
...,"Skye","Sloane","Smoky","Soleil","Sonia","Song","Sorority","Spice","Spider","Spring","Staci","Stacia","Starr","Stevie","Storm","Stormy","Strawberry","Suavecita","Sublime","Sugar","Summer","Sundae","Sundance","Sunday","Sunni","Sunny","Sunshine","Sunset","Susanna","Suzanne","Swallow","Sweet","Sweetness","Tabitha","Taffy","Talia","Tallulah","Tamar","Tamara","Tammi","Tane","Tanya","Tara","Tasha","Tasty","Tatiana","Tawni","Taylor","Temper","Tempest","Temptation","Terra","Terror","Tess","Tessa","Texxxas","Thai","Thia","Thumper","Thyme","Tia","Tickle","Tiffany","Tiger","Tigra","Tika","Time","Tina","Tinker","Tisha","Toffee","Tommy","Toni","Topaz","Tori","Traci","Tracy","Tricia","Trinity","Trip","Tristana","Trixie","Trouble","Tuesday","Tyffany","Tyler","Tyra","Unique","Utopia","Valentina","Valentine","Vandal","Vanessa","Vanity","Velvet","Venus","Veronica","Viagra","Victoria","Victory","Viola","Violet","Viper","Virginia","Virgo","Vision","Vivienne","Vixen","Vonda","Wanda","Wednesday","Wendy","Whimsy","Whisky","Whisper","Willow","Windy","Winter","Wish","Wyndi","Xanadu","Xanthe","Xaviera","Xena","Xiola","Xtase","Yasmin","Yolanda","Yvette","Yvonne","Zena","Zenith","Zinnia","Zoe","Zoey","Zora")
```
Comment (lordpidey) [inline code/modules/admin/verbs/one_click_antag.dm]: Good god, could you put this on multiple lines please?

Comment (Incoming5643) [inline code/modules/admin/verbs/one_click_antag.dm]: Absolutely not

**Pull request merged**